e2e/tests/host_controls.spec.ts

```diff
@@ -1,23 +1,35 @@
 import {expect, test} from '@playwright/test';
 
-import PlaywrightDevPage from '../page';
-import {getUsernamesForTest, getUserStoragesForTest, joinCall, startCall} from '../utils';
+import PlaywrightDevPage, {HostControlAction, HostNotice} from '../page';
+import {
+    getUsernamesForTest,
+    getUserStoragesForTest,
+    joinCall,
+    joinCallAndPopout,
+    startCall,
+    startCallAndPopout,
+} from '../utils';
 
 const userStorages = getUserStoragesForTest();
 const usernames = getUsernamesForTest();
 
+test.setTimeout(400000);
```

Comment:
This is a lot, can't be avoided I suppose?

Reply:
I noticed that the run time can vary really wildly. So most of the time it's quick, but sometimes it would take 5m for a run. Maybe it's a server load thing? I'd rather it not fail because of load.